src/shx.js

```diff
@@ -34,11 +34,16 @@ export const shx = (argv) => {
     return EXIT_CODES.SHX_ERROR;
   }
 
-  const ret = shell[fnName](...args);
-  const code = ret.hasOwnProperty('code') && ret.code;
+  var ret = shell[fnName](...args);
+  var code = ret.hasOwnProperty('code') && ret.code;
```

Comment:
I think we should use `let` here.  Const and let have advantages over var like block scoping, no hoisting, better semantics and proper write protection. 

If we're game for it, we can turn on eslint prefer const and no-var. This would ensure proper use of let/const everywhere.